packages/plugin-sass/src/index.ts

```diff
@@ -126,11 +126,6 @@ export const pluginSass = (
         .resolve.preferRelative(true)
         .end();
 
-      const inlineRule = chain.module
-        .rule(findRuleId(chain, CHAIN_ID.RULE.SASS_INLINE))
-        .test(include)
-        .resourceQuery(/inline/);
-
       // Support for importing raw Sass files
       chain.module
         .rule(CHAIN_ID.RULE.SASS_RAW)
```

Comment:
`CHAIN_ID.RULE.SASS_RAW` is `undefined` in v1.2.x. So we will get a rule like this:

```
      /* config.module.rule('undefined') */
      {
        test: /\.s(?:a|c)ss$/,
        resourceQuery: /raw/,
        type: 'asset/source'
      },
```

I'll make a patch in the next PR

```suggestion
        .rule(CHAIN_ID.RULE.SASS_RAW ?? "sass-raw")
```